### PR TITLE
various fix for metavisitor-2

### PR DESCRIPTION
**Pull request by drosofff**
- fix 5 min hits for blastn in 1-1, 1-2 and 1-3
- switch to trinityrnaseq for 3-3 because it manages better single end collections
- add trinityrnaseq in tool list (in addition to trinity from iuc)
- adapt testing in Docker (no install tools)

### CODE REVIEW

**Comment by drosofff** [comment]
see that @ccedmendoza

File: extra-files/metavisitor/Galaxy-Workflow-Metavisitor__Workflow_for_Use_Case_3-3.ga [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "a_galaxy_workflow": "true", 
     "annotation": "", 
     "format-version": "0.1", 
-    "name": "Metavisitor: Workflow for Use Case 3-3", 
+    "name": "Fix Metavisitor: Workflow for Use Case 3-3",
```
Comment (ccedmendoza) [inline extra-files/metavisitor/Galaxy-Workflow-Metavisitor__Workflow_for_Use_Case_3-3.ga]: we may need to change the title

Comment (drosofff) [reply extra-files/metavisitor/Galaxy-Workflow-Metavisitor__Workflow_for_Use_Case_3-3.ga]: Sure !

**Comment by drosofff** [comment]
:tada:  :confetti_ball:  :balloon: :tada:  :confetti_ball:  :balloon:
:tada:  :confetti_ball:  :balloon: :tada:  :confetti_ball:  :balloon:

**Pull request merged**